lnrpc/rpc.proto

```diff
@@ -1854,6 +1854,14 @@ message QueryRoutesRequest {
     zero, then the value of `--max-cltv-expiry` is used as the limit.
     */
     uint32 cltv_limit = 11;
+
+    /** 
+    An optional field that can be used to pass an arbitrary set of TLV records
+    to a peer which understands the new records. This can be used to pass
+    application specific data during the payment attempt. If the destination
+    does not support the specified recrods, and error will be returned.
+    */
+    map<uint64, bytes> dest_tlv = 13;
```

Comment:
On every other field that has the type `bytes` involved, we have the following comment:
> When using REST, this field must be encoded as base64.

Do we want to add this here too?

Reply:
Done

Reply:
To be consistent with the hop message, I would say this should be renamed to something like `dest_custom_records`. Comment should also mention custom record range